src/components/Table/readme.md

```diff
@@ -695,3 +695,216 @@ const tableContainerStyles = { height: 300 };
+    },
+    {
+        name: 'Sara',
+        company: 'nexxtway',
+        email: 'sara@gmail.com',
+        status: 'verified',
+        id: '5678qwerty',
+        link: '#',
+    },
+    {
+        name: 'Tahimi',
+        company: 'nexxtway',
+        email: 'tahimi@gmail.com',
+        status: 'verified',
+        id: '5678asdfgh',
+        link: '#',
+    },
+    {
+        name: 'Leo',
+        company: 'nexxtway',
+        email: 'leo@gmail.com',
+        status: 'verified',
+        id: '5678zxcvbn',
+        link: '#',
+    },
+    {
+        name: 'Tahimi Leon',
+        company: 'nexxtway',
+        email: 'leon@nexxtway.com',
+        status: 'verified',
+        id: '9012qwerty',
+        link: '#',
+    },
+    {
+        name: 'Alejandro',
+        company: 'Google',
+        email: 'alejandro@gmail.com',
+        status: 'verified',
+        id: '5678zdfgdf',
+        link: '#',
+    },
+    {
+        name: 'Carlos',
+        company: 'Oracle',
+        email: 'carlos@gmail.com',
+        status: 'verified',
+        id: '3434fgfgdf',
+        link: '#',
+    },
+    {
+        name: 'Luis',
+        company: 'Google',
+        email: 'luis@gmail.com',
+        status: 'verified',
+        id: '9012asdfgh',
+        link: '#',
+    },
+];
+
+const Title = styled.h1.attrs(props => props.theme.rainbow)`
+    font-family: 'Lato Light';
+    font-size: 25px;
+    text-align: left;
+    margin-bottom: 10px;
+    color: ${props => props.palette.text.main};
+`;
+
+const Subtitle = styled.h3.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 14px;
+    color: ${props => props.palette.text.title};
+`;
+
+const StyledLink = styled.a.attrs(props => props.theme.rainbow)`
+    color: ${props => props.palette.brand.main}
+    &:hover {
+        color: ${props => props.palette.brand.main};
+    }
+    &:visited {
+        color: ${props => props.palette.brand.main};
+    }
+    &:active {
+        color: ${props => props.palette.brand.main};
+    }
+`;
+
+const ProfileLink = ({value, row}) => <StyledLink href={row.link}>{ value }</StyledLink>;
+
+function TableExample (props) {
+    const [ data, setData ] = useState(props.data);
+    const [ sortedBy, setSortedBy ] = useState(undefined);
+    const [ sortDirection, setSortDirection ] = useState('asc');
+    const [ selection, setSelection ] = useState([]);
+
+    function handleOnSort(event, field, nextSortDirection) {
+        let newData = [...data];
+
+        const key = x => x[field];
+        const reverse = nextSortDirection === 'asc' ? 1 : -1;
+
+        const sortedData = newData.sort((a, b) => {
+            a = key(a);
+            b = key(b);
+            return reverse * ((a > b) - (b > a));
+        });
+
+        setData(sortedData);
+        setSortedBy(field);
+        setSortDirection(nextSortDirection);
+    }
+
+    function handleOnDeleteElements() {
+        const newData = data.filter( e =>  !(selection.map(x=>x.id).includes(e.id)));
```

Comment:
here you can put ```selection.map(x=>x.id)``` outside the filter in order to make the iteration only once, now it is iterating for each data element